### PR TITLE
Explicitly catch pkcon proceed prompt hit again as error

### PR DESCRIPTION
**Pull request by okurz**
Local verification run: http://lord.arch/tests/5474

Related progress issue: https://progress.opensuse.org/issues/15410

### CODE REVIEW

File: lib/susedistribution.pm [hunk 1]
```diff
@@ -134,7 +134,8 @@ sub ensure_installed {
     testapi::assert_script_sudo("chown $testapi::username /dev/$testapi::serialdev");
     my $retries = 5;    # arbitrary
     $self->script_run("for i in {1..$retries} ; do pkcon install $pkglist && break ; done ; RET=\$?; echo \"\n  pkcon finished\n\"; echo \"pkcon-\${RET}-\" > /dev/$testapi::serialdev", 0);
-    my @tags = qw/Policykit Policykit-behind-window pkcon-proceed-prompt pkcon-finished/;
+    my @tags        = qw/Policykit Policykit-behind-window pkcon-proceed-prompt pkcon-finished/;
```
Comment (richiejp) [inline lib/susedistribution.pm]: I thought we are using brackets now? :-)

Comment (okurz) [inline lib/susedistribution.pm]: not yet, only after my after PR is merged. I don't like to change the style on single lines when the focus should be on something else, in this case that I changed other parts of the code.

Comment (coolo) [inline lib/susedistribution.pm]: and I would like code touched to follow the style guide

Comment (okurz) [inline lib/susedistribution.pm]: done

**Comment by DimStar77** [comment]
> Local verification run: http://lord.arch/tests/5474

That test seems invalid: as the live image has no root password, you won't get a polkit auth dialog to ask for permission to install

https://openqa.opensuse.org/tests/321497#step/kate/9

Does not look like something would have gone wrong - except the sequence of seeing the old prompt again (still)

**Comment by chuchingkai** [comment]
prep_pidgin.pm use ensure_installed('pidgin') to install the pidgin, and it failed
due to this PR:

https://openqa.suse.de/tests/672071#step/prep_pidgin/10

**Comment by okurz** [comment]
on it

**Pull request merged**